templates/controllers/modals/submission/viewSubmissionMetadata.tpl

```diff
@@ -22,7 +22,7 @@
 					{if $metadataItem@iteration % 2 != 0}
 						<th scope="row">{$metadataItem}</th>
 					{else}
-						<td>{$metadataItem}</td>
+						<td>{$metadataItem|escape}</td>
```

Comment:
It looks like you've identified missing escaping here -- but if so, can you add it to the above `<th ...>` element as well?

Reply:
done, escaping the `<th ...>` as well now